static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -56,6 +56,14 @@ const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
             const adSlot = iframe && iframe.closest('.js-ad-slot');
+            if (adSlot.classList.contains('ad-slot--mostpop')) {
```

Comment:
IIRC `iframe.closest('.js-ad-slot')` will either return `Element` or `null`.

We probably want some way to handle that `null`:

```js
if (adSlot.classList && adSlot.classList.contains('ad-slot--mostpop')) {
    return Promise.resolve(false);
}
```